search/tutorials/buildsearchbar.mdx

```diff
@@ -0,0 +1,207 @@
+            </p>
+          </div>
+
+
+          {/* Search Component */}
+          <div className="max-w-2xl mx-auto">
+            <SearchComponent />
+          </div>
+
+
+          <div className="mt-20 grid md:grid-cols-3 gap-8">
+            <div className="text-center">
+              <div className="w-12 h-12 bg-blue-100 rounded-lg flex items-center justify-center mx-auto mb-4">
+                <svg className="w-6 h-6 text-blue-600" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2}
+                    d="M13 10V3L4 14h7v7l9-11h-7z" />
+                </svg>
+              </div>
+              <h3 className="text-lg font-semibold text-gray-900 mb-2">Lightning Fast</h3>
+              <p className="text-gray-600">Get instant search results powered by advanced indexing</p>
+            </div>
+
+            <div className="text-center">
+              <div className="w-12 h-12 bg-green-100 rounded-lg flex items-center justify-center mx-auto mb-4">
+                <svg className="w-6 h-6 text-green-600" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2}
+                    d="M9 12l2 2 4-4m6 2a9 9 0 11-18 0 9 9 0 0118 0z" />
+                </svg>
+              </div>
+              <h3 className="text-lg font-semibold text-gray-900 mb-2">Accurate Results</h3>
+              <p className="text-gray-600">Reranking ensures the most relevant content appears first</p>
+            </div>
+
+            <div className="text-center">
+              <div className="w-12 h-12 bg-purple-100 rounded-lg flex items-center justify-center mx-auto mb-4">
+                <svg className="w-6 h-6 text-purple-600" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+                  <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2}
+                    d="M12 6.253v13m0-13C10.832 5.477 9.246 5 7.5 5S4.168 5.477 3 6.253v13C4.168 
+                  18.477 5.754 18 7.5 18s3.332.477 4.5 1.253m0-13C13.168 5.477 14.754 5 16.5 
+                  5c1.746 0 3.332.477 4.5 1.253v13C19.832 18.477 18.246 18 16.5 18c-1.746 
+                  0-3.332.477-4.5 1.253" />
+                </svg>
+              </div>
+              <h3 className="text-lg font-semibold text-gray-900 mb-2">Comprehensive</h3>
+              <p className="text-gray-600">Search across all documentation, guides, and API references</p>
+            </div>
+          </div>
+        </div>
+      </main>
+
+    </div>
+  );
+}
+
+
+```
+
+---
+
+### 6. Start the Project
+
+Run the following command to start the development server:
+
+```bash
+npm run dev
+```
+
+Open your browser and navigate to `http://localhost:3000` to test the application.
+
+You can search through Upstash docs, and results will redirect you to the page you are looking for.
+
```

Comment:
Maybe a section about next steps or things to check, like docusaurus or search sdk docs